src/controllers/wishlist.controller.js

```diff
@@ -8,37 +8,40 @@ const addWishlist=async(req, res)=> {
     try {
         const token=req.header('Authorization').split(' ')[1]
         const userVerify=verfyToken(token,process.env.JWT_SECRET)
+        
         if (!req.user) {
             return res.status(401).json({ message: "unauthorized user!" });
         }
-        const { userId, productId } = req.body;
+       
+        const { productId } = req.body;
         const product = await Products.findByPk(productId);
-         const user = await User.findByPk(userId);
 
         if (!product) {
            return res.status(400).json({ message: "product not found" });
         }
```

Comment:
Try to use javascript in built in methods and functions instead of too much if conditions

Reply:
done